doc-content/enterprise-only/admin-and-config/managing-business-central-user-management-con.adoc

```diff
@@ -0,0 +1,18 @@
+[id='managing-business-central-user-management-con']
+= User management
+
+{CENTRAL} defines three types of entities for security management - users, groups, and roles. A role defines a set of permisions a user or group has in {CENTRAL}. {CENTRAL} provides the following roles:
```

Comment:
"A role defines..." => "Permissions can be assigned to both roles and groups in {CENTRAL}. {CENTRAL} provides the following roles:"

Reply:
Changed as suggested.